lib/charliecloud.py

```diff
@@ -259,7 +259,11 @@ def get(self):
          except KeyError:
             # Finally, prompt the user.
             # FIXME: This hangs in Bats despite sys.stdin.isatty() == True.
-            username = input("\nUsername: ")
+            try:
+               username = input("\nUsername: ")
+            except KeyboardInterrupt:
+               INFO('')
+               FATAL("keyboard interrupt")
```

Comment:
I'd recommend wording that's more user-facing, e.g. "cancelled" or something like that.